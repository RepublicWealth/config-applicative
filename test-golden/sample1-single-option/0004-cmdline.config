--- INI
--- ENV
--- CMDLINE
--XXX.abc "foo bar quux"
