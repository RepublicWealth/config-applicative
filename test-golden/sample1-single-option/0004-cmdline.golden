Parsed successfully:
Sample1 "foo bar quux"
